docs/tutorials/gettingstartedexample/gse-gen-basic.md

```diff
@@ -13,16 +13,126 @@
+
+    ![Create CSI-2 Packet TDMS Files Explorer View](../../images/PXIe-148X-CreateTDMS-ExplorerView.png)
+
+5. The newly created TDMS file will be used in the following tutorial to perform a simple generation.
 
 ## Performing a Simple Generation
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+
+1. Double click the Generation Example VI in the LabVIEW project.
+
+    ![Open Create CSI-2 Packet TDMS Files VI](../../images/PXIe-148X-GenGSE-Project.png)
+
+    > The opened front panel of the Generation Getting Started Example is similar to the figure below.
+
+    ![Generation GSE Front Panel](../../images/PXIe-148X-GenGSE-FrontPanel.png)
+
+
+2.  Select the **Resource** tab and make the following modifications.
+    - Select the **RIO Device** from the dropdown menu that corresponds to your interface module.
+    - Set the **Bitfile Path** to the bitfile that corresponds to your interface module. The default shipping bitfile is in the 'FPGA Bitfiles' subfolder located in the project folder (created at the Project Root location specified in Project Creator):
+
+    | **Interface Module**   | **Bitfile**                                          |
+    |------------------------|------------------------------------------------------|
+    | PXIe-1486 (8 Out)       | FPGA Bitfiles\\PXIe_1486_8\_Out.lvbitx              |
+    | PXIe-1486 (4 In 4 Out) | FPGA Bitfiles\\PXIe_1486_4\_In_4\_Out_Gen.lvbitx |
+    | PXIe-1487 (8 Out)       | FPGA Bitfiles\\PXIe_1487_8\_Out.lvbitx              |
+    | PXIe-1487 (4 In 4 Out) | FPGA Bitfiles\\PXIe_1487_4\_In_4\_Out_Gen.lvbitx |
+
+    > The values on the **Resource** tab of **Configuration Settings** are now similar to the figure below.
+
+    ![Configuration Settings Resource Tab](../../images/PXIe-148X-GenGSE-ResourceTab-BasicSimpleGen.png)
+
+3.  Run the VI to perform a generation using the TDMS file created in the previous tutorial.
+
+4.  The generation GSE example will first load the bitfile and then buffer available data from disk to onboard DRAM. After buffering, the **Waiting for Serializer Setup** indicator will enable to indicate the module is ready for generation.
+
+    ![Generation GSE Front Panel Waiting for Serializer Setup](../../images/PXIe-148X-GenGSE-FrontPanel-Waiting.png)
+
+5.  Click the **Serializer Setup Complete** control button to start generation. Generated images display automatically in the First Display Channel tab.
+
+    ![Configuration Settings Resource Tab](../../images/PXIe-148X-GenGSE-FirstDisplayChannel-BasicSimpleGen.png)
+
+6.  The VI will stop automatically when all data from the TDMS file has been generated.
```

Comment:
To use consistent present tense I suggest "The VI stops automatically after all data from the TDMS file is generated."